### ISSUE TITLE
Generated update SQL does NOT use backticks for reserved keywords

### ISSUE THREAD

**Issue by criuss** [go-jet/jet#57]
I am trying to do an update of a column with a reserved word as name in mysql.

Models und set value generate invalid SQL:
```go
type Model struct { ID: int, Load: string }
table.UPDATE(table.Load).MODELS(model{Load: "test"}).Where(...
```
AND
```go
table.UPDATE(table.Load).SET(mysql.String("test")).WHERE(...
```

generates:
```sql
UPDATE table
SET load = 'test'
WHERE ...;
```
set column with value generates valid SQL:
```go
table.UPDATE().SET(table.Load.SET(mysql.String("test"))).WHERE(...
```
generates the correct SQL:
```SQL
UPDATE uwaa.node_status
SET `load` = 'test'
WHERE ...;
```

**Comment by go-jet** [comment]
Looks like a bug.

**Comment by go-jet** [comment]
The bug is fixed in the [latests v2.5.0 release](https://github.com/go-jet/jet/releases/tag/v2.5.0).
